### ISSUE TITLE
Cross compile for mips architecture

### ISSUE THREAD

**Issue by FOTO-Miro** [hubaiz/DslrDashboardServer#37]
Hi
Do anybody have compiled ddserver for mips openwrt?
Please help it is very difficult for me.
Thanks

**Comment by hubaiz** [comment]
On what device do you use (maker/model)?

**Comment by FOTO-Miro** [comment]
Hi,

i use Space Station Mini from manufacturer wor(l)d

 

this is a battery mini mobile 4G router with Qualcomm MDM9600.

 

Thank you

 

Von: hubaiz ***@***.***> 
Gesendet: Dienstag, 26. April 2022 17:03
An: hubaiz/DslrDashboardServer ***@***.***>
Cc: FOTO-Miro ***@***.***>; Author ***@***.***>
Betreff: Re: [hubaiz/DslrDashboardServer] Cross compile for mips architecture (Issue #37)

 

On what device do you use (maker/model)?

**Comment by FOTO-Miro** [comment]
Hello,

do you have compiled ddserver fot the mips architecture ?

Greetings

 

Von: hubaiz ***@***.***> 
Gesendet: Dienstag, 26. April 2022 17:03
An: hubaiz/DslrDashboardServer ***@***.***>
Cc: FOTO-Miro ***@***.***>; Author ***@***.***>
Betreff: Re: [hubaiz/DslrDashboardServer] Cross compile for mips architecture (Issue #37)

 

On what device do you use (maker/model)?

**Comment by hubaiz** [comment]
No, not yet. Don't have currently a dev envrioment for OpenWrt build.

**Comment by FOTO-Miro** [comment]
ok

 

Von: hubaiz ***@***.***> 
Gesendet: Dienstag, 3. Mai 2022 16:49
An: hubaiz/DslrDashboardServer ***@***.***>
Cc: FOTO-Miro ***@***.***>; Author ***@***.***>
Betreff: Re: [hubaiz/DslrDashboardServer] Cross compile for mips architecture (Issue #37)

 

No, not yet. Don't have currently a dev envrioment for OpenWrt build.

**Comment by Roland0815** [comment]
Do you still need the ddserver.ipk for MIPS?

I just compiled ddserver_0.1-r1_mips_24kc.ipk and 
libusb-1.0-0_1.0.26-r3_mips_24kc.ipk 

with libstdcpp6_13.2.0-r4_mips_24kc.ipk 
and libatomic1_13.2.0-r4_mips_24kc.ipk

My TP-Link Archer C7 lost his old firmware. Was off now for over 1 or 2 years so my old precompiled OpenWRT with ddserver stopped working. 

I now installed openwrt-23.05.3-ath79-generic-tplink_archer-c7-v2-squashfs-sysupgrade and compiled the other .ipks. 

Need to test it in the next few days but installation worked fine.

**Comment by FOTO-Miro** [comment]
Hi, yes i still need it. How to download?
Thank you
